src/libs/actions/Report.js

```diff
@@ -1269,6 +1269,29 @@ function saveReportActionDraft(reportID, reportActionID, draftMessage) {
     Onyx.set(`${ONYXKEYS.COLLECTION.REPORT_ACTIONS_DRAFTS}${reportID}_${reportActionID}`, draftMessage);
 }
 
+/**
+ * Syncs up a chat report and an IOU report in Onyx after an IOU transaction has been made
+ * and they've both been updated in the back-end.
```

Comment:
Can you add a bit of the comment from [here](https://github.com/Expensify/Expensify.cash/pull/3544/files#diff-17f969ae17c2a0f476494263758dbe0a535432f496b805345925e648d605915cR206-R210)? I think this is where the main comment explanation should be, rather than duplicated in the two places it is called.